# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/crashreporter/CrashAnnotations.yaml
# Commit: bba32db4da1e
# Full Hash: bba32db4da1e4096b8286710b7f2a550796a2add
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-03-31 21:54:44
# Regressor Bug: 1586236
# File Overlap Count: 1
# Description:
#   Bug 1586236 - Use memory resource notifications to detect low memory scenarios on Windows. r=gsvelto
#   
#   This patch uses the low memory resource notification facility to detect
#   scenarios where physical memory is running low without polling. This is a
#   significant change compared to the previous behavior which measured both
# ==============================================================================

diff -r 1fc94d717bee -r bba32db4da1e toolkit/crashreporter/CrashAnnotations.yaml
--- a/toolkit/crashreporter/CrashAnnotations.yaml	Wed Mar 31 15:13:16 2021 +0000
+++ b/toolkit/crashreporter/CrashAnnotations.yaml	Wed Mar 31 15:18:05 2021 +0000
@@ -606,10 +606,10 @@
     mozilla::LauncherRegistryInfo::EnableState enum
   type: integer
 
-LowCommitSpaceEvents:
+LowPhysicalMemoryEvents:
   description: >
-    Number of times the available memory tracker has detected a that
-    commit-space was running low. This is a Windows-specific annotation.
+    Number of times the available memory tracker has detected that free
+    physical memory is running low. This is a Windows-specific annotation.
   type: integer
   ping: true
 